docs/how-tos/scaling.rst

```diff
@@ -0,0 +1,133 @@
+.. _scaling:
+
+Scaling your deployment
+***********************
+
+By default, the `Aspects Tutor plugin`_ deploys single nodes for these services:
+
+* :ref:`quick-start-ralph` or :ref:`quick-start-vector`
+* Clickhouse: ephemeral single node cluster
+* Superset: shares Open edX's MySQL and Redis instances
+
+Most deployments will benefit greatly from scaling horizontally and/or vertically, especially when running Aspects. Aspects is fed by Open edX event data, which is triggered by user actions on the site, and Open edX can generate a lot of event data. The initial processing of this event data occurs in platform plugins, so scaling your LMS workers speeds this processing time. And the faster events can be processed, the quicker you will see them appear in Aspects' dashboards and charts.
+
+And all production deployments will need a `persistent Clickhouse cluster`_.
+
+Preparing the LMS workers
+=========================
+
+Before deploying Aspects, we recommend :ref:`event_bus`.
+
+Or, if you'd rather use celery, we recommend boosting the number of LMS workers to prepare the celery queue for the high-volume of Aspects tasks.
+
+These plugins can be used to configure and enable autoscaling for the LMS and CMS. See their READMEs for details:
+
+* `tutor-contrib-pod-autoscaling`_ : for single instance deployments
+* `tutor-contrib-grove`_ : for multiple instance deployments
+
+We also recommend configuring the LMS to use the high-priority celery queue for Aspects tasks (`platform-plugin-aspects event sinks`_, `event-routing-backends xAPI tasks`_). This leaves the low-priority queue clear for other LMS tasks.
+
+.. code-block:: yaml
+
+  TBD -- implement configurable queue for Aspects tasks
+
+Scaling Ralph
+=============
+
+Without scaling, Ralph can be a bottleneck in the Aspects data pipeline. If you do not need an LRS at all, you can disable Ralph and use :ref:`quick-start-vector` instead.
+
+But if you will ever need an LRS, it's better to start with Ralph enabled and autoscaled.
+
+Ralph runs CPU-intensive operations, so we recommend scaling mainly on CPU:
+
+.. code-block:: yaml
+
+  TBD -- implement RALPH autoscaling
```

Comment:
cf https://github.com/openedx/openedx-aspects/issues/81

Reply:
@Ian2012 @bmtcril Should the scaling parameters for Ralph, Vector, and Superset be added to tutor-contrib-aspects, or should we keep them in https://github.com/eduNEXT/tutor-contrib-pod-autoscaling ?

Reply:
The values will be added via a filter in `tutor-contrib-aspects`. This will be once the next named release is public and this PR is released to the public: https://github.com/eduNEXT/tutor-contrib-pod-autoscaling/pull/7